api/core/core.proto

```diff
@@ -307,6 +307,8 @@ message LogEntry {
     string source    = 2;
     string level     = 3;
     string message   = 4;
+    // sorting key
```

Comment:
💡 Maybe we could remove this comment and just name the field `sortingKey`?

Reply:
done!
Thank you @opudrovs